script.js

```diff
@@ -1,7 +1,350 @@
+    tech: ['html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+    'link to live version': '#3',
+    'link to source': '#4',
+    class: "projects_art"
+  },
+
+  {
+    title: 'Data Dashboard Healthcare',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+    'link to live version': '#5',
+    'link to source': '#6',
+    class: "projects_data"
+  },
+
+  {
+    title: 'Website Portfolio',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    tech: ['html', 'bootstrap', 'Ruby'],
+    button: 'See Project',
+    'link to live version': '#7',
+    'link to source': '#8',
+    class: "projects_web"
+  },
+
+  {
+    title: 'Keeping track of hundreds of components website',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    tech: ['html', 'Bootstrap', 'Ruby on rails'],
+    'live button': 'See live',
+    'source button': 'See Source',
+    'image desktop': 'images/Snapshoot.png',
+    class: "hide"
+  },
+
+];
+
+const p1 = projects_container[0];
+const p2 = projects_container[1];
+const p3 = projects_container[2];
+const p4 = projects_container[3];
+const p5 = projects_container[4];
+
+// first mobile project
+const div21 = document.getElementById('ret');
+const img6 = document.createElement('img');
+img6.src = p1['image mobile'];
+img6.alt = 'alt';
+div21.appendChild(img6);
+
+const div22 = document.getElementById('works_mobile');
+const h38 = document.createElement('h3');
+h38.className = 'heed';
+h38.innerText = p1.title;
+div22.appendChild(h38);
+const para9 = document.createElement('p');
+para9.innerText = p1.description;
+div22.appendChild(para9);
+
+const div23 = document.getElementById('works2');
+const div24 = document.createElement('div');
+div24.className = 'sites';
+const tech9 = document.createElement('ul');
+for (const i of p1.tech) {
+  const li = document.createElement('li');
+  li.innerText = i;
+  tech9.appendChild(li);
+}
+div24.appendChild(tech9);
+div23.appendChild(div24);
+const button10 = document.createElement('button');
+button10.type = 'button';
+button10.className = 'button1';
+button10.innerText = p1.button;
+button10.onclick = function () { myFunction4(); };
+div23.appendChild(button10);
+
+// rendering mobile projects
+function renderProject(p) {
```

Comment:
- [ ] This issue hasn't been solved yet. The popup leads to the page to be stuck on the top part with the scroll disabled, one wouldn't access it. Please ensure to follow the figma design spec for this popup implementation.

Reply:
okay, I've enabled the scroll. Hope this resolves the issue.

Reply:
Not entirely, The popup needs to come to display from the same section of the project

Reply:
Please can you just jump on a quick zoom call and explain clearly what you're saying.